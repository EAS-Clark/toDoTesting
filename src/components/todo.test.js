import { render, screen } from '@testing-library/react';
import Todo from './todo';

test('renders learn react link', () => {
    // ({ text, state, toggleTodo })

    const toggleTodo = (index) => {
        setTodos(todos.map((todo, idx) => idx === index ? { ...todo.complete = !todo.complete } : todo));
    }
    const todo = {
        text: "Test input",
        state: false,
        toggleTodo: toggleTodo
    }
    render(<Todo todo />);

});

/*
<Todo {...todo} toggleTodo={toggleTodo} />

export const Todo = ({ text, state, toggleTodo }) => {
    return (
        <div className="todo">
            <p>{text}
                <input type="checkbox" checked={state} onChange={toggleTodo} />
            </p>
        </div>
    )
}
*/