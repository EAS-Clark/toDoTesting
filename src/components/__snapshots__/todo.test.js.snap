// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the same as the last Snapshot 1`] = `
<div
  className="todo"
>
  <p>
    <input
      onChange={[Function]}
      type="checkbox"
    />
  </p>
</div>
`;
